Core/Core/Views/CoreWebView.swift

```diff
@@ -249,6 +249,7 @@ open class CoreWebView: WKWebView {
                 checkSize()
                 document.addEventListener('load', checkSize, true)
             })
+            window.addEventListener('error', checkSize, true)
```

Comment:
if there is a broken image in the discussion the sizing wasn't quite right without this.